src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonClassInfo.cs

```diff
@@ -382,5 +361,51 @@ public bool DetermineExtensionDataProperty(Dictionary<string, JsonPropertyInfo>
 
             return converter;
         }
+
+        public bool DetermineExtensionDataProperty(Dictionary<string, JsonPropertyInfo> cache)
```

Comment:
Why were these methods moved (alphabetical sorting?)

Were there any changes (I didn't see any)?

Reply:
Arranged the methods mostly in the order that they are used from the constructor, but will revert as there's no real need to do this.